### ISSUE TITLE
RC.01: Shared color variables on default (light) and dark theme

### ISSUE THREAD

**Issue by ck2000** [coreui/coreui#110]
The default navbar, card and sidebar color vars are not used in the layout (see below). By default the e.g. $sidebar-**dark**-nav-color vars are used. This results in the problem that you can not define different colors for the light and dark theme for the nav. The default bootstrap 4 vars work fine.

```
// Navbar
$navbar-bg:                           $black !default;
$navbar-brand-bg:                     transparent !default;
$navbar-brand-minimized-bg:           $navbar-brand-bg !default;

$navbar-color:                        $gray-600 !default;
$navbar-hover-color:                  $gray-800 !default;
$navbar-active-color:                 $gray-800 !default;
$navbar-disabled-color:               $gray-300 !default;

// Sidebar Navigation
$sidebar-nav-color:       $black !default;
$sidebar-nav-title-color: $black !default;
$sidebar-nav-link-color: $black !default;
$sidebar-nav-link-icon-color:         $gray-600 !default;
$sidebar-nav-link-hover-color: $black !default;
$sidebar-nav-link-hover-bg:           theme-color("primary") !default;
$sidebar-nav-link-active-icon-color:  theme-color("primary") !default;
$sidebar-nav-dropdown-indicator-color: $gray-600 !default;
$sidebar-nav-dropdown-indicator-hover-color: $sidebar-nav-link-hover-color !default;
```

**Comment by mrholek** [comment]
I will fix it next release

**Comment by ck2000** [comment]
Thank you for the quick response!

I would be helpful to have two sperate color files e.g.:

- bootstrap-theme/dark-colors.scss
- bootstrap-theme/light-colors.scss (default) u name it ;)

**Comment by mrholek** [comment]
Fixed in v3.0.0-rc.3
